packages/flutter/test/widgets/visibility_test.dart

```diff
@@ -310,4 +432,458 @@ void main() {
 
     semantics.dispose();
   }, skip: isBrowser);
+
+  testWidgets('SliverVisibility', (WidgetTester tester) async {
```

Comment:
nit: I would probably put these in their own sliver_visiblity_test.dart file.

Reply:
SGTM!